### DISCUSSION TITLE
Vercel + Sveltekit not working

### DISCUSSION THREAD

**Discussion by samfcheng** [sveltejs/kit#8962] [Q&A] [answered]
Looks like #8902 could have/did create a breaking bug when trying to deploy to Vercel.
![image](https://user-images.githubusercontent.com/51756184/217709581-cacb482a-c65f-4cf4-b7e9-8f8da63d03c2.png)

Originally thought it was a Vercel issue but [turns out it's not](https://github.com/vercel/community/discussions/1459)

How can I solve this?

**Comment by teemingc** [comment, marked as answer]
You'll either need to upgrade @sveltejs/kit to version 1.5.0 or upgrade adapter-auto to 2.0.0.

There was a recent update to all adapters that required a minimum version of sveltekit. However, adapter-auto would still downloaded the latest adapter versions without checking the sveltekit version. This update to adapter-auto fixes that.
